moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -305,6 +305,13 @@ void MotionPlanningDisplay::reset()
   query_robot_goal_->setVisible(query_goal_state_property_->getBool());
 }
 
+void MotionPlanningDisplay::setName(const QString& name)
+{
+  BoolProperty::setName(name);
```

Comment:
@Jntzko  it looks like the reason why ```BoolProperty::setName(name)``` works is because the **MotionPlanningPanel** inherits from **rviz::Display** which itself inherits from **BoolProperty** so this is not a static method but a super class call.  The inherited ```setName()``` method is already implemented by the [rviz::Display](http://docs.ros.org/indigo/api/rviz/html/c++/display_8cpp_source.html#l00410) class and it already does call the same functions that you have added.  Since the **MotionPlanningPanel** already has inherits this method from rviz::Display then there should be no need to override it.